commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -5,6 +5,64 @@
 @import views.html.hosted.guardianHostedCta
 
 @main(page, Some("commercial")) { } {
+    <!--[if (gt IE 9)|(IEMobile)]><!-->
```

Comment:
Good point! I will add this to the TODO list and implement it as one of the last step. Currently it's just a proof of concept to test this solution.